h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -220,6 +218,7 @@ public AutoML(Key<AutoML> key, Date startTime, AutoMLBuildSpec buildSpec) {
     super(key);
 
     this.startTime = startTime;
+    this.runCountdown = new Countdown(Math.round(1000 * buildSpec.build_control.stopping_criteria.max_runtime_secs()));
```

Comment:
perhaps it would be better to keep the countdown in seconds instead of ms?

Reply:
I prefer to keep it in ms, there's information loss when converting ms to s, so I don't want this to be done implicitely: for example 50ms could be converted either to 0s or 1s, and if 0s, then the semantic changes (here it's interpreted as no timeout like almost everywhere in our code).

The best I can do is provide a static factory method maybe: `Countdown.fromSeconds()`